### PR TITLE
Refactor dApp Structure 05 | More mvd cleanup

### PR DESCRIPTION
**Pull request by Da-Colon**
## Description

Moved more of the smaller changes out of the Fractal Provider update PR. so the PR isn't difficult to review

### CODE REVIEW

File: src/components/DaoCreator/provider/CreatorProvider.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ export const initialState: CreatorState = {
   step: CreatorSteps.ESSENTIALS,
   prevStep: null,
   nextStep: null,
-  governance: GovernanceTypes.TOKEN_VOTING_GOVERNANCE,
+  governance: GovernanceTypes.GNOSIS_SAFE,
```
Comment (herbig) [inline src/components/DaoCreator/provider/CreatorProvider.tsx:24]: it seems strange to me that this changed from `TOKEN_VOTING_GOVERNANCE` to `GNOSIS_SAFE`.  Is this changing from MVD 1:1 to Gnosis multisig?

**Review by herbig — COMMENTED** [review]
Have a question about one line change, but otherwise looks gooood

File: src/components/DaoCreator/provider/CreatorProvider.tsx (hunk 1 above)
Comment (herbig) [reply src/components/DaoCreator/provider/CreatorProvider.tsx:24]: ah ok ok, yeah sounds good

**Review by herbig — APPROVED** [review]

**Review by tbwebb22 — APPROVED** [review]
LGTM

**Pull request merged**